frontend/src/metabase/common/hooks/use-collection-query/use-collection-query.ts

```diff
@@ -0,0 +1,18 @@
+import Collections from "metabase/entities/collections";
+import { Collection, CollectionId } from "metabase-types/api";
+import {
+  useEntityQuery,
+  UseEntityQueryProps,
+  UseEntityQueryResult,
+} from "../use-entity-query";
+
+export const useCollectionQuery = (
+  props: UseEntityQueryProps<CollectionId, unknown>,
+): UseEntityQueryResult<Collection> => {
+  return useEntityQuery(props, {
```

Comment:
I didn't add any test, should I?

Reply:
Use sure, add a test similar to other entity hooks to make sure it works end-to-end

Reply:
added